pkg/config/validation/validation.go

```diff
@@ -3363,10 +3363,6 @@ var ValidateServiceEntry = registerValidateFunc("ValidateServiceEntry",
 					}
 				}
 			}
-			if serviceEntry.Resolution == networking.ServiceEntry_DNS_ROUND_ROBIN && len(serviceEntry.Endpoints) != 1 {
```

Comment:
Do not we need this validation if endpoints are specified? i.e. if they are specified - should n't it be exactly one?

Reply:
Why cannot it have two?

Reply:
Does envoy deny it?

Reply:
yes. It NACKs it with this message ""LOGICAL_DNS clusters must have a single locality_lb_endpoint and a single lb_endpoint"

Reply:
0 is allowed as well as 1. I can change this.

Reply:
I thought it'd be best to revert and come up with a fix. But I can do both.